src/Tizen.NUI.Wearable/src/public/Popup.cs

```diff
@@ -0,0 +1,524 @@
+﻿/*
+ * Copyright(c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Tizen.NUI.BaseComponents;
+using Tizen.NUI.Components;
+
+namespace Tizen.NUI.Wearable
+{
+    using L = Tizen.Log;
+
+    /// <summary>
+    /// Popup is an UI component to give a notification or message, which interaction with user simply.
+    /// It is attached to Window directly so that it is shown on top of all UI components.
+    /// Title(text) and Content(container) are initially formed and user can control them by setting properties or styles.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class Popup : Control
```

Comment:
Are there many differences from NUI.Component.Popup?

Reply:
I tried to change this popup to be inherited from Components.Popup. However, it has several things to be discussed. 

1. Components.Popup uses ButtonGroup and it's visual update is done in ButtonGroup class, and Popup also has private method to update it's own visuality. To make this enable, they should be changed as virtual methods and it looks not very intuitive to override two depth of instance. 

2. AddButton() interface doesn't have the positioning or anchoring of button, so it can't be overrided this method. also this method will be appeared by inheriting, so that users will be confused. 

3. It is possible to implement by inheriting Popup, but the implementation will be overlapped and it seems to have no merit to inherit.